cnf-certification-test/accesscontrol/suite.go

```diff
@@ -561,44 +559,29 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
 	testhelper.AddTestResultLog("Non-compliant", podsWithRestrictedTolerationsNotDefault, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
 
-const (
-	listProcessesCmd     = "ls -l /proc/*/exe"
-	sshDaemonProcessName = "sshd"
-)
-
 func TestNoSSHDaemonsAllowed(env *provider.TestEnvironment) {
+	// The command prints yes to stdout in case sshd is running. ps is not
+	// available on ubi-minimal, iterates over /proc.
+	const isSshdCmd =
+		"ls -l /proc/*/exe 2>/dev/null | grep -q sshd && printf yes"
 	var badContainers []string
 	var errorContainers []string
-
 	o := clientsholder.GetClientsHolder()
-	r := regexp.MustCompile(sshDaemonProcessName)
-
 	for _, cut := range env.Containers {
-		stdout, stderr, err :=
-			o.ExecCommandContainer(
-				clientsholder.NewContext(cut.Namespace, cut.Podname, cut.Name),
-				listProcessesCmd,
-			)
-		if err != nil || stderr != "" {
-			var msg string
-			switch {
-			case stderr == "":
-				msg = fmt.Sprintf("error: %v", err)
-			case err == nil:
-				msg = fmt.Sprintf("stderr: %s", stderr)
-			default:
-				msg = fmt.Sprintf("error: %v, stderr: %s", err, stderr)
-			}
-			tnf.ClaimFilePrintf("Unable to list processes at %s, %s.", cut, msg)
+		out, _, err := o.ExecCommandContainer(
```

Comment:
If the `stderr` will be ignored anyway we might as well keep the former implementation.

Reply:
I've approved your PR with minor changes. You can merge it.